src/ripple/app/misc/ValidatorList.h

```diff
@@ -399,36 +400,35 @@ ValidatorList::onConsensusStart (
         }
     }
 
-    // This quorum guarantees sufficient overlap with the trusted sets of other
-    // nodes using the same set of published lists.
-    std::size_t quorum = keyListings_.size()/2 + 1;
-
-    // Increment the quorum to prevent two unlisted validators using the same
-    // even number of listed validators from forking.
-    if (localPubKey_.size() && ! localKeyListed &&
-            rankedKeys.size () > 1 && keyListings_.size () % 2 != 0)
-        ++quorum;
+    // This minimum quorum of 2f + 1 guarantees safe overlap with the trusted
```

Comment:
I'd rather we don't mention `f` at all here: 

    // The minimum quorum guarantees safety with up 1/3 of the listed
    // validators being malicious.